source/MaterialXRuntime/Commands/AttributeCommands.h

```diff
@@ -3,8 +3,8 @@
 // All rights reserved.  See LICENSE.txt for license.
 //
 
-#ifndef MATERIALX_RTCONNECTIONCOMMANDS_H
-#define MATERIALX_RTCONNECTIONCOMMANDS_H
+#ifndef MATERIALX_RTATTRIBUTECOMMANDS_H
```

Comment:
Not really related to this review, but I noticed this guard wasn't named correctly.